packages/ceramic-cli/src/index.ts

```diff
@@ -104,24 +111,57 @@ program
   })
 
 program
-  .command('change <docId> [content]')
-  .option('--owners <owners>', 'Change owner of this document (only 3ID)')
-  .description('Update the content of a document')
-  .action(async (docId, content, { owners }) => {
+  .command('pin_add <docId>')
+  .description('Pin the document locally')
```

Comment:
Looking at the api here we have:
* `ceramic pin_add <docId>`
* `ceramic pin_rm <docId>`
* `ceramic pin_list [<docId>]`

Can we change it to this? (might be a bit more difficult, but will be much nicer :) 
* `ceramic pin add <docId>`
* `ceramic pin rm <docId>`
* `ceramic pin ls [<docId>]`